test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs

```diff
@@ -222,14 +282,30 @@ private static HandleModelAttributesContext GetDefaultContext()
 
         #region Test Data
 
-        private class MarkdownModel1
+        public class LoopModel
+        {
+            [MarkdownContent]
+            public string Content { get; set; }
+
+            [UniqueIdentityReference]
+            public string Identity { get; set; }
+
+            public LoopModel Reference { get; set; }
+        }
+
+        public class MarkdownModel1
         {
             [MarkdownContent]
             public string Content { get; set; }
 
             public string Content2 { get; set; }
 
-            public List<string> Content3 { get; set; }
+            [MarkdownContent]
+            public string ReadonlyContent { get; } = "*content";
+
+            [UniqueIdentityReference]
+            [UniqueIdentityReferenceIgnore]
```

Comment:
To test that UniqueIdentityReferenceIgnore takes effect over UniqueIdentityReference